src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -65,31 +67,46 @@ class Tasks(Enum):
     PR_SENSITIVITY = auto()
 
 
+@contextmanager
```

Comment:
nit: make private and move to bottom of file

Reply:
Good catch, accepted!